cap/jsonschemas/definitions/alice-datasets-v0.0.1.json

```diff
@@ -33,7 +33,7 @@
       "title": "Dataset AOD",
       "type": "string"
     },
-    "run_numbers": {
+    "run_number": {
```

Comment:
Generally, it would be preferable to have the same term across schemas. So, if we go with singular, it will match the rest of the schemas, where this is being used. If plural is better, we can keep this as it was and I will just change the others to plural as well.
Same goes for dataset/datasets below. So far, the vast majority of schemas use the singular.